src/Resources/config/storage/doctrine.xml

```diff
@@ -16,5 +16,11 @@
             <argument type="string">TaskBundle:TaskExecution</argument>
         </service>
         <service id="task.storage.task_execution" alias="task.repository.task_execution"/>
+
+        <service id="task.event_listener.execution" class="Task\TaskBundle\EventListener\TaskExecutionListener">
```

Comment:
If this service is loaded or not should be configureable. It might be possible that the `EntityManager` should be cleared although the tasks are stored somewhere else.